backend/modules/forum/posts/delete/dto/delete.args.ts

```diff
@@ -0,0 +1,8 @@
+import { ArgsType, Field, Int } from "@nestjs/graphql";
+
+@ArgsType()
+export class DeletePostsForumsArgs {
+
+  @Field(() => Int)
+  post_id: number;
```

Comment:
```suggestion
  id: number;
```
You have name as `forum_posts__delete` mutation. You don't have to duplicate `post`.